src/main/java/com/coveros/selenified/element/check/Equals.java

```diff
@@ -469,22 +469,22 @@ default String checkSelectedValue(String expectedValue, double waitFor, double t
      * @return String[]: all of the select values of the element. null will be returned if the element isn't present or a select
      */
     @SuppressWarnings("squid:S1168")
-    default String[] checkSelectValues(String[] expectedValues, double waitFor, double timeTook) {
+    String[] checkSelectValues(String[] expectedValues, double waitFor, double timeTook) {
         // record the action, and check it's a select
         if (!isPresentSelect(
-                getElement().prettyOutput() + " with select values of <b>" + Arrays.toString(expectedValues) +
+                this.element.prettyOutput() + " with select values of <b>" + Arrays.toString(expectedValues) +
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal " with select values of <b>" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)